packages/logging/src/entry.js

```diff
@@ -32,7 +32,7 @@ var is = require('is');
  * @alias module:logging/entry
  * @constructor
  *
- * @param {object=|string=} resource - See a
+ * @param {object=} metadata - See a
  *     [Monitored Resource](https://cloud.google.com/logging/docs/api/ref_v2beta1/rest/v2beta1/MonitoredResource).
```

Comment:
Is this the correct URL still? Might be this instead/in addition:
https://cloud.google.com/logging/docs/api/ref/rest/v1beta3/projects.logs.entries/write#LogEntryMetadata